velox/functions/prestosql/StringFunctions.h

```diff
@@ -341,6 +341,57 @@ struct StrLPosFunction : public StrPosFunctionBase<T, true> {};
 template <typename T>
 struct StrRPosFunction : public StrPosFunctionBase<T, false> {};
 
+/// hamming_distance(string, string) -> bigint
+/// Computes the hamming distance between two strings.
+template <typename T>
+struct HammingDistanceFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  template <typename TCodePoint>
+  void doCall(
```

Comment:
make this private

Reply:
rewrite little bit, so this function is not needed.